src/constatnts.py

```diff
@@ -11,14 +11,14 @@
 ITER_AS_SEC = 20
 
 TOP_COORD = 30.24268
-LEFT_COORD = -88.77964
 DOWN_COORD = 30.19767
+LEFT_COORD = -88.77964
 RIGHT_COORD = -88.72648
 
 POINT_LAT_SIZE = (TOP_COORD - DOWN_COORD)/POINTS_SIDE_COUNT
 POINT_LON_SIZE = (RIGHT_COORD - LEFT_COORD)/POINTS_SIDE_COUNT
 
-POINT_LAT_CENTERS = [DOWN_COORD + POINT_LAT_SIZE/2 + (POINT_LAT_SIZE * i) for i in range(POINTS_SIDE_COUNT)]
+POINT_LAT_CENTERS = [TOP_COORD + POINT_LAT_SIZE/2 - (POINT_LAT_SIZE * i) for i in range(POINTS_SIDE_COUNT)]
```

Comment:
For example, when point_lat_size is 2, first centre would be 31.24 which is out of the map. Previous version look ok

Reply:
first sign should be - rather than + <- that should help

still previous versions was not right as the top coordinate has 0 index, so should get  TOP_COORD  center rather than DOWN_COORD